cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,49 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+let user;
+let article;
+
+describe('Conduit', () => {
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.visit('');
```

Comment:
The `cy.visit('');` call is missing a URL. You need to specify the URL of the page you want to visit for the test to run correctly.